adapter/redisquota/driver.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2017 Google Inc.
```

Comment:
The code in here needs to be made thread-safe. 

The various methods of an adapter instance can be called multiple times concurrently so the driver code can also be as a result.